examples/balanced_lineupsNBA.py

```diff
@@ -0,0 +1,118 @@
+"""
+Generate as many lineups as you want with
+max exposures of your selected exposure to any one player,
+with a specified minimum average of your choice.
+The output here is a CSV file that can be used
+to upload lineups at https://www.draftkings.com/lineup/upload
+
+Pre-reqs:
+1) Salary file exists at passed location
+2) Projection file exists at passed location
+3) PIDS file downloaded from aforementioned location
+
+Example usage:
+```python
+from examples import balanced_lineupsNBA
```

Comment:
Same comment on the file name as on the pid file name. Probably most pythonic to do `balanced_lineups_nba.py` (and would be great to change the NFL one too). I'm also okay if you just remove this comment since it's duplicating the NFL one, and the README has some instructions on how to do this.